### PR TITLE
Issue 259: create CocoaPods task for Calypso

### PR DESCRIPTION
**Pull request by artdaw**
#259: create CocoaPods task for Calypso

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -183,7 +183,7 @@ GEM
       fastlane_core (>= 0.44.0)
       slack-notifier (~> 1.3)
       terminal-table
-    xcpretty (0.2.2)
+    xcpretty (0.2.1)
```
Comment (bonkey) [inline Gemfile.lock]: Why downgrade? ;-)

File: lib/calypso/pod.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+    desc 'lint_lib', 'Lint Library'
+    def lint_lib
+      run 'pod lib lint AtlasSDK.podspec --allow-warnings'
+    end
+
+    desc 'release', 'Release new version: create new tag and push to the GitHub'
+    def release
+      current_version = read_current_version
+      say "Pleae, add new AtlasSDK version (current #{current_version}):", :blue
```
Comment (bonkey) [inline lib/calypso/pod.rb]: Typo: `Pleae`

File: lib/calypso/pod.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+      new_version = ask 'Please, enter new version', :blue
+      write_new_version(new_version)
+
+      invoke :lint_lib
+
+      run "git add -A && git commit -m 'Release #{new_version}.'"
+      run "git tag '#{new_version}'"
+      run "git push --tags"
```
Comment (bonkey) [inline lib/calypso/pod.rb]: I'm a bit nervous with raw git commands, maybe default to just print it, and you can copy and paste it?

File: lib/calypso/pod.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
+require 'thor'
+require_relative 'run'
+
+FILE_NAME = 'version.rb'
```
Comment (bonkey) [inline lib/calypso/pod.rb]: `VERSION_FILE`?

File: lib/calypso/pod.rb [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      if File.exists?(FILE_NAME)
+          file_str = File.open(FILE_NAME, 'r') {|file| file.read }
+      end
+
+      return file_str.match(/[0-9\.]+/)[0]
+    end
+
+    def write_new_version(new_version)
+      file_str = ''
+      if File.exists?(FILE_NAME)
```
Comment (bonkey) [inline lib/calypso/pod.rb]: Is it required to check it? Should be enough to just `.write...`

**Comment by bonkey** [comment]
👍

**Comment by bonkey** [comment]
And drop version.rb to `lib` folder

File: Gemfile.lock (hunk 1 above)
Comment (artdaw) [inline Gemfile.lock]: Fixed

File: lib/calypso/pod.rb (hunk 4 above)
Comment (artdaw) [inline lib/calypso/pod.rb]: renamed to VERSION_FILE

File: lib/calypso/pod.rb (hunk 2 above)
Comment (artdaw) [inline lib/calypso/pod.rb]: Fixed

**Comment by bonkey** [comment]
👍

**Comment by artdaw** [comment]
👍

**Pull request merged**